eth-connector/src/lib.rs

```diff
@@ -547,8 +547,8 @@ impl EngineConnectorWithdraw for EthConnectorContract {
 
 #[near_bindgen]
 impl Deposit for EthConnectorContract {
-    fn deposit(&mut self, #[serializer(borsh)] raw_proof: Proof) -> Promise {
-        self.connector.deposit(&raw_proof)
+    fn deposit(&mut self, #[serializer(borsh)] proof: Proof) -> Promise {
+        self.connector.deposit(proof)
```

Comment:
Why we should borrow value here?

Reply:
As I can see here, we move ownership of the `proof` into the `deposit` method. It's ok.